src/coverlet.collector/coverlet.collector.csproj

```diff
@@ -18,6 +18,8 @@
     <EnablePackageValidation>true</EnablePackageValidation>
     <!-- disable transitive version update and use versions defined in coverlet.core -->
     <CentralPackageTransitivePinningEnabled>false</CentralPackageTransitivePinningEnabled>
+    <!-- create SBOM -->
+    <GenerateSBOM>true</GenerateSBOM>
```

Comment:
does it make more sense to only do this in a release build?

Reply:
I think generate SBOM generate only for release build is not sufficient. This shall be done for CI builds and not for local builds.